math/check_prime.cpp

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2020 @author omkarlanghe
+ * Reduced all possibilities of a number which cannot be prime.
+ * Eg: No even number, except 2 can be a prime number, hence we will increment our loop with i+2 jumping on all odd numbers only.
+ * If number is <= 1 or if it is even except 2, break the loop and return false telling number is not prime.
+ */
+#include <iostream>
+
+ /**
+ * Function to check if the given number is prime or not.
+ * @param no number to be checked.
+ * @return if number is prime, it returns @ true, else it returns @ false.
+ */
+int is_prime(int no) {
```

Comment:
instead of `no` we can use `num` or just `n`, i think this looks more readable isn't it ?